### DISCUSSION TITLE
Manually Trigger Disabled Workflow

### DISCUSSION THREAD

**Discussion by AlexSneedMiller** [community/community#31366] [Actions]
We have a GitHub Action Workflow set up to deploy our code to our TEST environment. This Workflow is triggered when a PR is merged. When we want to 'freeze' our environment so that we can perform additional testing without the environment changing we disable the workflow.

When doing bug fixes, we often will then enable the workflow and manually re-run it. 

Is it possible to manually run the workflow without enabling it altogether so that it doesn't run on a PR merge?

**Comment by Simran-B** [comment]
Would removing whatever `on` event you currently use and replacing it with `workflow_dispatch` be an option?
It would only run when triggered manually, so there would be no need to disable the workflow anymore.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
